content/docs/rfcs/template/README.md

```diff
@@ -5,18 +5,19 @@ name: (Waku v2 | Logos | Codex) RFC Template
 status: (raw|draft|stable)
 category: (Standards Track|Informational|Best Current Practice)
 tags: an optional list of tags, not standard
+description: an optional short summary
 editor: Daniel Kaiser <danielkaiser@status.im>
 contributors:
```

Comment:
> Could you add GitHub user names for editors and contributors as well?
> This https://github.com/status-im/status-website/blob/e84d7266a170bf13e23f548add5ddb6b3d069de6/content/help/communities/about-community-request-approvals.mdx#L6 is how [@]campo's docs' team write their frontmatter. [Could be a joint effort and a some combination of two]
>
> [...]
>
> ok will do. Also, is there way to find github usernames that match the status.im email address?
>
> https://contacts.status.im/
>
> If you'd like to automate it or script it, DevOps ([@]jakubgs) might have a way/endpoint/even a list from past.
>
> [...]
>
> there are a couple emails that are not status, in spec 24 and 28
>
> I'd leave those as-is, hence making the gh username optional. But will leave the decision up to you guys (Vac team).
Or to get the username from a git commit 🤔.
>
> – https://discord.com/channels/864066763682218004/1179553924617871422/1179763113499840513

Reply:
@felicio moving this conversation to https://github.com/vacp2p/rfc/pull/644